test/literate_tabbed.litcoffee

```diff
@@ -69,13 +70,15 @@ if true
 
 	test "should ignore code blocks inside HTML", ->
 		eq executed, false
+		testsCount++
 
 ---
 
-*   A list item with a code block:
+*   A list item followed by a code block:
 
-		test "basic literate CoffeeScript parsing", ->
-			ok yes
+	test "basic literate CoffeeScript parsing", ->
```

Comment:
But that breaks the spec. Shouldn't it be easier to just ban nested code block? Since there's no way to write one that don't break the spec, while keeping the indentation consistent.